Fluid/Filters/MiscFilters.cs

```diff
@@ -842,5 +847,77 @@ public static ValueTask<FluidValue> Sha256(FluidValue input, FilterArguments arg
                 return new StringValue(builder.ToString());
             }
         }
+
+        /// <summary>
+        /// https://shopify.dev/api/liquid/filters/additional-filters#default_pagination
+        /// </summary>
+        public static ValueTask<FluidValue> DefaultPagination(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            if (input.ToObjectValue() is not PaginateStatement.PaginateValue paginate) return StringValue.Empty;
+
+
+            using (var sb = StringBuilderPool.GetInstance())
+            {
+                var builder = sb.Builder;
+
+                if (paginate.Previous?.IsLink == true)
+                {
+                    var title = paginate.Previous.Title;
+
+                    if (arguments.HasNamed("previous"))
+                    {
+                        var str = arguments["previous"].ToStringValue();
+                        if (!string.IsNullOrWhiteSpace(str)) title = str;
+                    }
+
+                    builder.AppendFormat(
+                            "<span class=\"prev\"><a href=\"{0}\">{1}</a></span>",
```

Comment:
Any piece of text that will be rendered and could be customized should go in options. There could be a sub-object called `PaginateSettings` with many things to change.

Then it's accessible here from `context.Options.PaginateSettings`.

Reply:
Maybe we can create a delegate to implement this function. This requires the PaginateValue and PartValue to be exposed.